src/api/cli/integration/integration_test.go

```diff
@@ -1341,3 +1339,311 @@ func TestLiveWalletDir(t *testing.T) {
+	w, totalCoins, _ := prepareAndCheckWallet(t, 2, 16)
+
+	tt := []struct {
+		name    string
+		args    func() []string
+		errMsg  []byte
+		checkTx func(t *testing.T, txid string)
+	}{
+		{
+			// Send all coins to the first address to one output.
+			"send all coins to the frist address",
+			func() []string {
+				coins, err := droplet.ToString(totalCoins)
+				require.NoError(t, err)
+				return []string{"send", w.Entries[0].Address.String(), coins}
+			},
+			nil,
+			func(t *testing.T, txid string) {
+				// Confirms all coins are in the first address in one output
+				tx := getTransaction(t, txid)
+				require.Len(t, tx.Transaction.Transaction.Out, 1)
+				c, err := droplet.FromString(tx.Transaction.Transaction.Out[0].Coins)
+				require.NoError(t, err)
+				require.Equal(t, totalCoins, c)
+			},
+		},
+		{
+			// Send 0.5 coin to the second address.
+			// Send 0.5 coin to the third address.
+			// After sending, the first address should have at least 1 coin left.
+			"send to multiple address with -m option",
+			func() []string {
+				addrCoins := []struct {
+					Addr  string `json:"addr"`
+					Coins string `json:"coins"`
+				}{
+					{
+						w.Entries[1].Address.String(),
+						"0.5",
+					},
+					{
+						w.Entries[2].Address.String(),
+						"0.5",
+					},
+				}
+
+				v, err := json.Marshal(addrCoins)
+				require.NoError(t, err)
+
+				return []string{"send", "-m", string(v)}
+			},
+			nil,
+			func(t *testing.T, txid string) {
+				tx := getTransaction(t, txid)
+				// Confirms the second address receives 0.5 coin and 1 coinhour in this transaction
+				checkCoinsAndCoinhours(t, tx, w.Entries[1].Address.String(), 500000, 1)
+				// Confirms the third address receives 0.5 coin and 1 coinhour in this transaction
+				checkCoinsAndCoinhours(t, tx, w.Entries[2].Address.String(), 500000, 1)
+				// Confirms the first address has at least 1 coin left.
+				coins, _ := getAddressBalance(t, w.Entries[0].Address.String())
+				require.True(t, coins >= 1000000)
+			},
+		},
+		{
+			// Send 0.001 coin from the third address to the second address.
+			// Set the second as change address, so the 0.499 change coin will also be sent to the second address.
+			// After sending, the second address should have 1 coin and 1 coin hour.
+			"send with -c(change address) -a(from address) options",
+			func() []string {
+				return []string{"send", "-c", w.Entries[1].Address.String(),
+					"-a", w.Entries[2].Address.String(), w.Entries[1].Address.String(), "0.001"}
+			},
+			nil,
+			func(t *testing.T, txid string) {
+				tx := getTransaction(t, txid)
+				// Confirms the second address receives 0.5 coin and 0 coinhour in this transaction
+				checkCoinsAndCoinhours(t, tx, w.Entries[1].Address.String(), 500000, 0)
+				// Confirms the second address have 1 coin and 1 coin hour
+				coins, hours := getAddressBalance(t, w.Entries[1].Address.String())
+				require.Equal(t, uint64(1000000), coins)
```

Comment:
1e6 notation and I don't think you need to explicitly wrap it around uint64 here